pages/Guides/supernode-program/docs.en.md

```diff
@@ -6,92 +6,122 @@ taxonomy:
 
   Opening port 7880 is required to allow the servant to work correctly.
 
-  Port 7778 allows light wallets to connect to the server.
+  Port 7778 allows NEM NanoWallet to connect using websockets to the server.
 
-- **Step 9**: Start NIS, let it synchronize and then start the servant.
+- **Step 10**: Start NIS Client, Let it Synchronize, and Then Start the Servant.
 
-  To start the servant double click on ``runservant.bat`` in the ``servant`` folder (In Mac or Linux navigate to the file in terminal and use the command ``sh startservant.sh``). If it works you will see your terminal/command prompt window become active.
+  To start the servant double-click on ``runservant.bat`` in the ``servant`` folder. (In Mac or Linux navigate to the file in terminal and use the command ``sh startservant.sh``. It's recommended to run in background using ``screen`` or ``nohup`` command.)
 
-- **Step 10**: Send an unencrypted enrollment message to the official node rewards account.
+- **Step 11**: Enroll in the Program.
 
-  Send a NEM message to ``NAFUNDBUKIOSTMD4BNXL7ZFE735QHN7A3FBS6CMY`` stating "enroll (your IP or hostname) (your nodes name) (your delegated harvesting public key)" from the primary account that has the 3 million XEM deposit to officially announce you would like to participate in the program. To find the delegated public key, open your 3 million XEM account in NCC. Go to *View Account Details* and then you can see the delegated account's public key. Please note that this is **not** the delegated harvesting **private** key you used earlier.
+  Go to Services -> SuperNode Program -> Check & Enroll in Program -> Enroll in Program, put the enrollment address for current month (announced via [Twitter](https://twitter.com/nemofficial) and [Discord](https://www.discord.gg/xymcity)) and host for your NEM node. Send enrollment transaction.
 
-  ![find delegated public key](delegated-key.png)
+  ![NEM NanoWallet Enrollment](wallet-enroll.png)
 
-  ![Example for an enrollment message](enroll-1.png)
+  Make sure host matches your node host from ``/node/info`` endpoint response.
 
-- **Step 11**: Review your results at [supernodes.nem.io](http://supernodes.nem.io/) to make sure that your node is passing all the tests. Please be patient, since your results will not be visible immediately.
+  ![Endpoint response](node-info.png)
 
-  For more discussion about this blog, please visit the [NEM Forum](https://web.archive.org/web/20210814074405/https://forum.nem.io/t/nem-supernode-rewards-program/1735).
+  The Supernode Program requires monthly re-enrollments. We believe the set-and-forget philosophy of node operation rewards unhealthy behaviours, especially in a decentralised network where validators need to make informed decisions pertaining to the state of the network.
```

Comment:
I feel monthly re-enrollment deserves its own section; not directly related to the initial configuration. I'd put it in at the very bottom

```
## Monthly Re-enrollment
The Supernode Program requires monthly re-enrollments. We believe the set-and-forget philosophy of node operation rewards unhealthy behaviours, especially in a decentralised network where validators need to make informed decisions pertaining to the state of the network.
Prior to the month-end, a new enrollment address will be announced by [Twitter](https://twitter.com/nemofficial) and [Discord](https://www.discord.gg/xymcity).
The node operator has to re-enroll each month using a new enrollment address. Enrollment for the next month is available 4 days before the end of the preceding month.

Refer to Step 11 in [Steps to Enroll and Participate in the Supernode Program](#Steps-to-Enroll-and-Participate-in-the-Supernode-Program)
```

-- I'd also check the link at the bottom, as I'm still getting the hang of markup

Reply:
Moved with correct linking :+1: